numpy/distutils/command/config.py

```diff
@@ -441,7 +441,7 @@ def get_output(self, body, headers=None, include_dirs=None,
                 src, obj, exe = self._link(body, headers, include_dirs,
                                            libraries, library_dirs, lang)
                 grabber.restore()
-            except:
+            except Exception:
                 output = grabber.data
                 grabber.restore()
                 raise
```

Comment:
This one might have been correct as is... I'm not really sure what's going on here :-), but in general bare `except` followed by bare `raise` is an acceptable pattern.